eng/versioning/version_client.txt

```diff
@@ -49,6 +49,7 @@ com.azure:azure-communication-chat;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-communication-common;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-communication-administration;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-communication-sms;1.0.0-beta.3;1.0.0-beta.4
+com.azure:azure-communication-identity;1.0.0-beta.3;1.0.0-beta.4
```

Comment:
do we have to put `1.0.0-beta.3` for identity? The first release for identity would be `beta.4`

Reply:
Removed `1.0.0-beta3`